spec/unit/puppet/provider/cert_file/posix_spec.rb

```diff
@@ -7,7 +7,7 @@
 require 'openssl'
 describe 'The POSIX provider for type cert_file' do
   before do
-    test_keys = OpenSSL::PKey::RSA.new(2049)
```

Comment:
2049 bits was very specific. I wonder if that was intentional.

Reply:
Probably a typo.

Note: using short keys in unit tests when a lot of keys have to be generated can speed up the tests significantly.  Is it worth here?